tracing-subscriber/src/fmt/mod.rs

```diff
@@ -116,7 +116,7 @@
 //! [`EnvFilter`]: ../filter/struct.EnvFilter.html
 //! [`env_logger`]: https://docs.rs/env_logger/
 //! [`filter`]: ../filter/index.html
-//! [`fmtBuilder`]: ./struct.SubscriberBuilder.html
+//! [`fmtBuilder`]: ./struct.CollectorBuilder.html
 //! [`FmtSubscriber`]: ./struct.Collector.html
```

Comment:
this link reference looks like it's for a link that either _has_ changed or needs to change, is that link broken or does it still say `Subscriber`?

Reply:
whoops, needed to fix the _remaining_ documentation in this section. done.